README.rst

```diff
@@ -19,3 +15,77 @@ to the `pretalx configuration documentation`_ for further details.
 .. _ansible-galaxy: https://galaxy.ansible.com/
 .. _here: https://github.com/pretalx/ansible-pretalx/blob/master/defaults/main.yml
 .. _pretalx configuration documentation: https://docs.pretalx.org/en/latest/administrator/configure.html
+
+Quick Start
+-----------
+
+You will need an Ansible playbook that uses this role, a database for Pretalx to connect to, and Nginx to front the Python server. 
+
+You can configure variables in that playbook, overriding anything in the defaults_.
+
+The following steps were tested on Debian 10:
```

Comment:
Could you also add the Ansible and pretalx versions you used for testing, please? Ansible versions tend to have horrible backwards compatibility, and if this guide is prominent in the README (which is great btw, thank you!), it should be dated to avoid future frustration.

Reply:
I did not know that about Ansible! Added.